pyproject.toml

```diff
@@ -64,6 +74,8 @@ markers = [
     "redisjson: only redisjson tests",
     "redistimeseries: only redistimeseries tests",
     "redisbloom: only redisbloom tests",
+    "redisai: only redisai tests",
+    "redisaidag: only redisai tests with dag",
```

Comment:
Why separate these out? Curious

Reply:
because there are a lot of dag tests, and I thought they deserve a tag